python/kfserving/README.md

```diff
@@ -1,6 +1,28 @@
-# KFServing
+# KFServing Python SDK
+Python SDK for KFServing Server and Client.
 
-KFServing is a unit of model serving. KFServing's python libraries implement a standardized KFServer library that is extended by model serving frameworks such as XGBoost and PyTorch. It encapsulates data plane API definitions and storage retrieval for models.
+## Installation
+
+KFServing Python SDK can be installed by `pip` or `Setuptools`.
+
+### pip install
+
+```sh
+pip install kfserving
+```
+
+### Setuptools
+
+Install via [Setuptools](http://pypi.python.org/pypi/setuptools).
+
+```sh
+python setup.py install --user
+```
+(or `sudo python setup.py install` to install the package for all users)
+
+
+## KFServing Server
+KFServing is a unit of model serving. KFServing's python libraries implement a standardized KFServing library that is extended by model serving frameworks such as XGBoost and PyTorch. It encapsulates data plane API definitions and storage retrieval for models.
```

Comment:
"KFServing is a unit of model serving." is an awkward statement here.

Reply:
Updated, thanks.